node/topAccountListener_test.go

```diff
@@ -284,14 +284,19 @@ func TestInit(t *testing.T) {
 
 func makeBlockWithTxnFor(senders []byte, receivers []byte) bookkeeping.Block {
 	var blk bookkeeping.Block
+	blk.BlockHeader.GenesisID = "foo"
+	blk.BlockHeader.GenesisHash[0] = 3
```

Comment:
I'm not sure why you need to make the GenesisID/GenesisHash.. but there is nothing wrong with these changes.

Reply:
it would generally be better if you were to pick a genesisHash that is really pseudo-random, i.e. crypto.Hash([]byte{1,2,3,4})

Reply:
GenesisID and GenesisHash must the same in transactions and block header. Why is pseudo-random better?

Reply:
a true random means that the result of the test might be random.. and we don't want that.
filling up only selected bytes doesn't guarantee that the comparison was testing all the bytes.
(i.e. both me and you knows it does.. but we want to ensure we test that )

Reply:
Thanks. Done.